editoast/assets/sprites/generate-atlas.sh

```diff
@@ -1,4 +1,8 @@
-#!/bin/sh
```

Comment:
Maybe we want to stay POSIX compatible? If I understand correctly, the move to `bash` is about the `globstar` option? You might be able to change the `cp` command with something along these lines:

```sh
find "${signaling_system}" -type f -name "*.svg" -exec cp "{}" "${svg_dir}" ";"
```

Reply:
Sorry, I got mixed up in my versions.

By switching to spreet, we now have smaller png, and it can crawl subdirectories, so no need to switch to 15 year young stuff, and no need for a syntax that always needs to be googled ;)